src/main/java/com/spotify/futures/CompletableFutures.java

```diff
@@ -66,6 +66,26 @@ private CompletableFutures() {
   }
 
   /**
+   * Returns a new {@link CompletableFuture} which completes to a list of values of those input
+   * stages that succeeded. The list of results is in the same order as the input stages. For failed
+   * stages, the defaultValueMapper will be called, and the value returned from that function will
+   * be put in the resulting list.
+   *
+   * <p>If no stages are provided, returns a future holding an empty list.
+   *
+   * @param stages The stages to combine.
```

Comment:
Documented `defaultValueMapper` and fixed style. `<T>` is never documented, however.